From f4037861ca7fc995d556d891d0520d0bb365e193 Mon Sep 17 00:00:00 2001
From: Christian Kellner <christian@kellner.me>
Date: Mon, 19 Jun 2017 13:11:19 +0200
Subject: [PATCH] Use qmake-qt5 as qmake binary

---
 qrenderdoc/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/qrenderdoc/CMakeLists.txt b/qrenderdoc/CMakeLists.txt
index b6863a2..214a886 100644
--- a/qrenderdoc/CMakeLists.txt
+++ b/qrenderdoc/CMakeLists.txt
@@ -199,7 +199,7 @@ endforeach()
 # which causes CMake to rerun this target every time so that Qt
 # can do dependency checking and rebuild anything necessary.
 add_custom_command(OUTPUT QRenderDoc
-	COMMAND qmake "CMAKE_DIR=${CMAKE_BINARY_DIR}" ${CMAKE_CURRENT_SOURCE_DIR}
+	COMMAND qmake-qt5 "CMAKE_DIR=${CMAKE_BINARY_DIR}" ${CMAKE_CURRENT_SOURCE_DIR}
 	COMMAND $(MAKE)
 	DEPENDS RenderDoc.icns
 	DEPENDS ${swig_output})
-- 
2.9.4

